spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationを設定したことによって`project_id`の指定が必要なくなると思います。
```suggestion
  let(:task) { create(:task) }
```

Reply:
2648bd0 でproject_idの記述を削除しました。